### PR TITLE
[#176091543] Add PagoPaGetUser function

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

File: PagoPAGetUser/handler.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          _1 => taskEither.of(maybeNoticeEmail),
+          _1 => taskEither.of(maybeNoticeEmail)
+        );
+      });
+    return (enableNoticeEmailCache
+      ? tryCatch(
+          () => sessionStorage.getPagoPaNoticeEmail(user),
+          _ => new Error("Error reading the notify email cache")
+        )
+          .foldTaskEither(_ => fromLeft<Error, EmailString>(_), fromEither)
```
Comment (balanza) [inline PagoPAGetUser/handler.ts]: I think this is enough:
```suggestion
        ).map(fromEither)
```

Comment (BurnedMarshal) [reply PagoPAGetUser/handler.ts]: This line is needed to handle the `Either` on the right and merge the two left scenarios on a single path.

Comment (balanza) [reply PagoPAGetUser/handler.ts]: sorry, actually it was `chain`

```suggestion
        ).chain(fromEither)
```

this works, same result

Comment (BurnedMarshal) [reply PagoPAGetUser/handler.ts]: Nice, it works fine. Fixed in 15a4a73

**Pull request merged**